internal/infrastructure/cache/cache_benchmark_test.go

```
@@ -0,0 +1,217 @@
+
+	// Measure performance of JSON operations
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		jsonData, _ = json.Marshal(lease)
+
+		c.Set(fmt.Sprintf("json-lease-%d", i), jsonData, time.Hour)
+
+		value, _ = c.Get(fmt.Sprintf("json-lease-%d", i))
+		retrievedData = value.([]byte)
+
+		json.Unmarshal(retrievedData, &retrievedLease)
```

golangci-lint (1.64.8)

Error return value of `json.Unmarshal` is not checked

(errcheck)